lib/ViewModels/TerriaViewer.js

```diff
@@ -303,6 +309,29 @@ function changeFogSettings(viewer) {
   }
 }
 
+function changeQuality(viewer) {
+  if (defined(viewer.terria.cesium)) {
+    const terria = viewer.terria;
+
+    // 0 - defaults
+    // 1 - devicePixelRatio as resolutionScale
+    // 2 - increase maximumScreenSpaceError
+    switch (viewer.terria.quality) {
+      case 1:
+        terria.cesium.viewer.resolutionScale = devicePixelRatio;
+        terria.cesium.viewer.scene.globe.maximumScreenSpaceError = 1;
+        break;
+      case 2:
+        terria.cesium.viewer.scene.globe.maximumScreenSpaceError = 4.0 / 3.0;
```

Comment:
Set `resolutionScale` here too, otherwise it won't happen if the user clicked on the right directly rather than dragging.